src/gerbil/runtime/build.scm

```diff
@@ -25,7 +25,28 @@
     (if (not (zero? (process-status proc)))
       (error "Compilation error; gsc exit with nonzero status" modf))))
 
+(define (update-gx-version)
+  (define gx-version-path
+    "gx-version.scm")
+  (define git-version
+    (if (file-exists? "../../../.git")
+      (let* ((proc (open-process '(path: "git" arguments: ("describe" "--tags") show-console: #f)))
```

Comment:
you should check that the exist status, as reported by process-status, is actually zero here.